infra/base-images/base-builder/write_labels.py

```diff
@@ -0,0 +1,17 @@
+#!/usr/bin/python3
+
+import os
+import json
+import sys
+
+def main():
+  tags = json.loads(sys.argv[1])
```

Comment:
make add a message if sys.argv len is < 2, and exit with some exit code.

Reply:
Done.